scripts/install-windows-prereqs.ps1

```diff
@@ -401,27 +401,32 @@ function Install-7Zip {
 }
 
 function Install-PSW {
+
+    $OS_VERSION = Get-WindowsRelease
     $tempInstallDir = "$PACKAGES_DIRECTORY\Intel_SGX_PSW"
     if(Test-Path $tempInstallDir) {
         Remove-Item -Recurse -Force $tempInstallDir
     }
     Install-ZipTool -ZipPath $PACKAGES["psw"]["local_file"] `
                     -InstallDirectory $tempInstallDir
-
-    $installer = Get-Item "$tempInstallDir\Intel*SGX*\PSW_EXE*\Intel(R)_SGX_Windows_x64_PSW_*.exe"
-    if(!$installer) {
-        Throw "Cannot find the installer executable"
-    }
-    if($installer.Count -gt 1) {
-        Throw "Multiple installer executables found"
-    }
-    $unattendedParams = @('--s', '--a', 'install', "--output=$tempInstallDir\psw-installer.log", '--eula=accept', '--no-progress')
-    $p = Start-Process -Wait -NoNewWindow -FilePath $installer -ArgumentList $unattendedParams -PassThru
-    if($p.ExitCode -ne 0) {
-        Get-Content "$tempInstallDir\psw-installer.log"
-        Throw "Failed to install Intel PSW"
+    if ($OS_VERSION -eq "WinServer2016") {
+        $installer = Get-Item "$tempInstallDir\Intel*SGX*\PSW_EXE*\Intel(R)_SGX_Windows_x64_PSW_*.exe"
+        if(!$installer) {
+            Throw "Cannot find the installer executable"
+        }
+        if($installer.Count -gt 1) {
+            Throw "Multiple installer executables found"
+        }
+        $unattendedParams = @('--s', '--a', 'install', "--output=$tempInstallDir\psw-installer.log", '--eula=accept', '--no-progress')
+        $p = Start-Process -Wait -NoNewWindow -FilePath $installer -ArgumentList $unattendedParams -PassThru
+        if($p.ExitCode -ne 0) {
+            Get-Content "$tempInstallDir\psw-installer.log"
+            Throw "Failed to install Intel PSW"
+        }
+    } else {
+        $psw_dir = Get-Item "$tempInstallDir\Intel*SGX*\PSW_INF*\"
+        pnputil /add-driver $psw_dir\sgx_psw.inf /install
```

Comment:
Do we need to catch any errors from `pnputil`?

Reply:
When it fails it will throw an error. I went down the path of error catching and retrying but as we are deprecating the scripts eventually just got it to work and made a note about it in the PR.